Open_Source_Events/GSSOC/README.md

```diff
@@ -0,0 +1,6 @@
+# GirlScript Summer Of Code
+<p align="center">
+<img src="https://user-images.githubusercontent.com/68437435/110908762-493a6080-8335-11eb-8dd8-a0f184767fd3.png">
```

Comment:
Put the image at End of the File after the text

Reply:
OK